elevenlabs/api/voice.py

```diff
@@ -68,6 +68,8 @@ class VoiceDesign(API):
     name: str
     text: str = Field(..., min_length=100)
     gender: Gender
+    description: str
+    labels: dict
```

Comment:
Does `labels` take any value as keys or is it looking for specific keys?

Reply:
labels use specific keys being age, gender, and accent